src/views/portal/AcceptedMessage.tsx

```diff
@@ -0,0 +1,139 @@
+import styles from "views/portal/Portal.module.scss";
+
+function AcceptedMessage() {
+	return (
+		<>
+			<h4>Waiver</h4>
+			<p>
+				ALL ATTENDEES, please review our{" "}
+				<a href="https://docs.google.com/document/u/1/d/e/2PACX-1vRPuYBKQ2YR64FVzDKunDeVkeFGD573hSqfTmf8JwvyhlvPQPEnovxhlYbal45WI3k3Jr2iLNRzz1Hp/pub">
+					Code of Conduct
+				</a>{" "}
+				and sign the Event Waiver prior to the event. You will receive the
+				waiver via email closer to the event date.{" "}
+				<strong className={styles.red_text}>
+					<i>
+						<u>
+							You must complete the waiver in order to participate in Hack at
+							UCI.
+						</u>
+					</i>
+				</strong>
```

Comment:
Thought: The underline might be a bit excessive (I think a good practice is reserving the use of underline for hyperlinks)